plugins/more_selectors/more_selectors.py

```diff
@@ -0,0 +1,201 @@
+import cadquery as cq
+from math import sqrt
+from .utils import make_debug_cylinder, make_debug_sphere
+
+#These function only works when they are in the run file
+"""
+def make_debug_cylinder(plane, outer_radius, inner_radius=None, height = None):
```

Comment:
There is no difference, I should remove the ones commentend here, but these function and the debug option of the selectors doesn't work since it use show_object method from cq-editor. I think it's a nice feature for visual debugging but I don't know how to make it work the way I want, if you have any suggestion i'm open about it. We can also simply remove them if you think its best. I opened an issue in cq-editor about it : https://github.com/CadQuery/CQ-editor/issues/254#issue-836241163